examples/custom-node/src/evm/env.rs

```diff
@@ -14,7 +14,10 @@ use reth_ethereum::evm::{primitives::TransactionEnv, revm::context::TxEnv};
 ///
 /// [`Evm`]: alloy_evm::Evm
 #[derive(Clone, Debug)]
-pub struct CustomEvmTransaction(pub OpTransaction<CustomTxEnv>);
+pub enum CustomEvmTransaction {
```

Comment:
we should maybe come up with some consistent naming, and name this CustomTxEnv?

can do this separately